pkg/storagecapabilities/storagecapabilities.go

```diff
@@ -89,7 +89,7 @@ var CapabilitiesByProvisionerKey = map[string][]StorageCapabilities{
 	"pxd.openstorage.org/shared": createOpenStorageSharedVolumeCapabilities(),
 	"pxd.openstorage.org":        createRWOBlockAndFilesystemCapabilities(),
 	"pxd.portworx.com/shared":    createOpenStorageSharedVolumeCapabilities(),
```

Comment:
Noticed that we are saying RWX/Block is supported but https://github.com/libopenstorage/openstorage/blob/master/csi/controller.go#L408-L412 begs to differ?

Reply:
Seems we may need to revamp and correct the portworx caps and backport to previous releases too.

Reply:
Ok, so the code impliciltly says RWX/File,RWO/Block,RWO/File are supported in the shared provisioners?